cc/crypto/client_encryptor.h

```diff
@@ -56,8 +56,8 @@ class ClientEncryptor {
   // <https://datatracker.ietf.org/doc/html/rfc5116>
   //
   // Returns an [`oak.crypto.EncryptedRequest`] proto message.
-  absl::StatusOr<oak::crypto::v1::EncryptedRequest> Encrypt(absl::string_view plaintext,
-                                                            absl::string_view associated_data);
+  absl::StatusOr<::oak::crypto::v1::EncryptedRequest> Encrypt(absl::string_view plaintext,
```

Comment:
I thought in google3 the preference is not to fully qualify, unless necessary to compile: http://go/totw/151

Reply:
We have an internal namespace that ends with `oak`, and our external namespace starts with `oak`. So if we use a library from Github internally, it tries to find names in the internal `oak::crypto::v1` which doesn't exist.